test/unit/sql/create-table.test.js

```diff
@@ -27,6 +27,31 @@ describe(Support.getTestDialectTeaser('SQL'), function() {
         });
       });
     });
+    if(Support.getTestDialect() === 'postgres') {
+      describe('offers Postgres support', function() {
```

Comment:
Should I rename to `Postgres IF NOT EXISTS version check`, as its a Postgres only check